### ISSUE TITLE
Warlock fear

### ISSUE THREAD

**Issue by Kenniflopper** [gamefreedomgit/Maelstrom#3714]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
Warlock fear damage threshold is bugged, it breaks in 1 second on a target with dots  which is absurdly bugged.

**How to reproduce:**
Pop demon soul, on use vicious badge and fear a target and see how fast it breaks. 

**How it should work:**
There is like a threshold the fear can take before it breaks, right now it breaks almost instantly

**Database links:**

**Comment by Kenniflopper** [comment]
Atleast in arenas it breaks instantly

**Comment by Pornoodemon** [comment]
Fear sometimes goes astray with one-two tick of Corruption or from one cast Immolate

**Comment by Pornoodemon** [comment]
This source states that fear should disappear when taking damage equal to 40% of base health.
And also the last changes in the mechanics of fear were in patch 3.2.2
![image](https://github.com/gamefreedomgit/Maelstrom/assets/133828891/f2490cc1-4af5-488c-99f0-8945250ac342)
https://www.mmo-champion.com/threads/711283-The-History-of-Fear



![image](https://github.com/gamefreedomgit/Maelstrom/assets/133828891/ce78bbe9-37ca-4ee7-9717-3de3904526f9)

40% base health on cataclysm is about 15-20k hp. And this damage threshold is already relatively close to the truth

**Comment by Pornoodemon** [comment]
besides, the big question is how this threshold damage should worked with the effects of absorbing damage

**Comment by UltraNix** [comment]
Right now it's 10%.

**Comment by UltraNix** [comment]
https://www.mmo-champion.com/threads/1244552-Dmg-threshold-thats-why-fear-wont-break
They are also not shure about the thresold.

**Comment by Kenniflopper** [comment]
Heartbeat mechanic was removed when cc duration went from 10sec to 8 sec in pvp, its not in cataclysm.

**Comment by UltraNix** [comment]
@Kenniflopper Do you have any sources? I still see some special attribute for these spells.
Also, the 10% damage thresold is correct.

**Comment by UltraNix** [comment]
It's classic, so I would not take this into account.

**Comment by UltraNix** [comment]
I will increase damage thresold to 20%.

**Comment by Kenniflopper** [comment]
Okay but i know they removed heartbeat resist from the game before cata, idk why it would be in the files.

**Comment by UltraNix** [comment]
Sorry - but recently I've fixed many bugs that in fact were not bugs (and reporters provided no sources). So I will wait here about that heartbeat mechanic for more info.

**Comment by Kenniflopper** [comment]
I mean u can increase the threshold and close it, i cant find any proper evidence from 2011-2012 about heartbeat.

**Comment by UltraNix** [comment]
Fixed.

**Comment by mipanka** [comment]
https://youtu.be/rPXue4sObxg?t=200 rogue take ~10k dmg only dots and fear was break

**Comment by Kenniflopper** [comment]
Yeah so i was mistaken and u gotta revert that fix you did and everything will be back to normal again :D
